site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Setup manageJobsWithoutQueueName"
+title: "Manage Jobs without queue name"
```

Comment:
Actually, I would suggest moving the content about defaulting LocalQueues out to a separate subpage. Sibling to this one. The reason being that the defaulting is also relevant for VAP, so there is no reason to couple it to manageJobsWithoutQueueName . The title could be "Setup LocalQueue defaulting".

Reply:
I think currently default local queue feature related to how manage jobs without queue name and it would make sense to have all settings in one place, so users will understand how those feature work together and what the difference between them. 

However what is VAP? Maybe it worth to add the link to default LocalQueue section in other places.

Reply:
https://github.com/kubernetes-sigs/kueue/blob/main/site/content/en/docs/tasks/manage/enforce_job_management/setup_job_admission_policy.md

Reply:
> I think currently default local queue feature related to how manage jobs without queue name and it would 
> make sense to have all settings in one place, so users will understand how those feature work together and 
> what the difference between them.

I would rather link the dedicated page from both subpages for manageJobsWithoutQueueName and VAP. Since the defaulting happens in webhook it applies to both features. "have all settings in one place" - yeah in the recent update of docs on main the three related features would be under the common folder `enforce_job_management`.

Reply:
For better visibility of the feature we may also link it from https://kueue.sigs.k8s.io/docs/reference/labels-and-annotations/#kueuex-k8sioqueue-name

Reply:
Done